src/train_model.py

```diff
@@ -169,13 +170,26 @@ def train_model(config: Struct):
             plugins=[SLURMEnvironment(requeue_signal=signal.SIGHUP)],
             callbacks=[early_stopping, model_checkpoint],
             logger=tb_logger)
-
+        
+    ## Set up carbon emissions tracker
+        
+    CO2_csv_path = config.CO2_outfile if config.CO2_outfile else None
+
+    emissions_tracker = OfflineEmissionsTracker(
+                output_dir=model_dir,
+                output_file=CO2_csv_path,
+                country_iso_code="USA",
```

Comment:
May want to consider adding these as parameters to the yaml for reproducibility once we submit the code, but for now I think this is fine for our own uses.

Reply:
I like the thought of that. I don't want to complicate our yaml file too much, I'd rather it not be intimidating to other users, but this will likely bite them if we bury it in our code instead of the yaml. Lets make note of that for later.